wgsl/index.bs

```diff
@@ -4660,8 +4668,12 @@ textureSampleLevel(t : texture_depth_cube_array, s : sampler, coords : vec3<f32>
   <tr><td>`offset`<td>
   The optional texel offset applied to the unnormalized texture coordinate
```

Comment:
sounds like we should have a single place where texture offset semantics is described and just reference it in all these cases instead of repeating ourselves

Reply:
Maybe. As a reader of specs, I dislike having to jump around the document to get all the facts (a bit like why I overlooked the need for offsets to be compile time constant!). I appreciate that updating the same text over and over is not fun though and may be prone to mistakes.

Reply:
It works both ways though. If you are looking  for it, you'll find it faster when it's repeated. If you aren't looking for it and just scrolling around, passing through the same text makes it less of a pleasure to scroll. Overall, given that the description spans multiple paragraph, it seems worthy moving it away. We'll discuss this on the editor call (want to join?)

Reply:
Actually, leaving this to WGSL spec editors

Reply:
Let's leave it duplicated for now, so the info is available in the place you need it. If it becomes an issue, we can clean it up later.